docs/ai/quickstarts/snippets/local-ai/ollama.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.AI.Ollama" Version="9.5.0-preview.1.25265.7" />
```

Comment:
A general note that this package is also referenced by name in the local AI quickstart content itself, outside the code snippets. I'm not sure off hand if others do as well.